js/src/dropdown.js

```diff
@@ -159,6 +161,12 @@ const Dropdown = (($) => {
             element = parent
           }
         }
+        // If boundary is not `scrollParent`, then set position to `static`
+        // to allow the menu to "escape" the scroll parent's boundaries
+        // https://github.com/twbs/bootstrap/issues/24251
+        if (this._config.boundary !== 'scrollParent') {
+          $(parent).addClass('position-static')
```

Comment:
Maybe you can add `position-static` to our const var `ClassName` and replace that line by : 
```js
$(parent).addClass(ClassName.POSITION_STATIC)